x-pack/plugin/esql/compute/test/build.gradle

```diff
@@ -3,5 +3,6 @@ apply plugin: 'elasticsearch.build'
 dependencies {
   api project(':test:framework')
   api project(xpackModule('esql:compute'))
+  api project(xpackModule('esql-core'))
```

Comment:
This is like requiring `ql` here. Just to double check, is this absolutely necessary?

I know that we rely on this already in `compute`, so I assume the answer is "yes". But still wanted to double check :)

Reply:
It's required, yeah. I'll leave a comment about what we use it for. I think, ideally, we wouldn't need it and the comment can help readers in six months know.